src/components/Header/Header.js

```diff
@@ -1,17 +1,38 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
 import PropTypes from 'prop-types';
 
-import { I18N } from '../../utils/constants';
-
 import style from './Header.module.css';
 
 const Header = (props) => {
+  const { t, i18n } = useTranslation();
   const { isAuth, login } = props;
+
+  const activateRu = (e) => {
+    i18n.changeLanguage('ru');
+  };
+
+  const activateEn = (e) => {
+    i18n.changeLanguage('en');
+  };
+
   return (
     <header className={style.header}>
+      <button onClick={(e) => activateRu(e)}>RU</button>
+      <button onClick={(e) => activateEn(e)}>EN</button>
       <div className={style.loginBlock}>
-        {isAuth ? login : <NavLink to="/login">{I18N.EN.LOGIN}</NavLink>}
+        {isAuth
+          ? (
+            <div>
+              {login}
+              {' '}
```

Comment:
Is there a way to avoid such empty space string?
E.x. using <br/> tags

Reply:
yep